spec/models/reservations_validations_spec.rb

```diff
@@ -24,4 +24,24 @@
     reservation.update_attributes reserve_start_at: now, reserve_end_at: now + 1.hour + 5.minutes
     expect(reservation.errors[:base]).to be_present
   end
+
+  describe "starts_before_cutoff" do
+    context "when there is a cutoff" do
+      before :each do
+        reservation.product.update_attribute(:cutoff_time, 2)
```

Comment:
We should probably add a validation on instrument for numericality equal or greater than 0.